Owncloud iOs Client/Tabs/FileTab/DeleteFile/DeleteFile.m

```diff
@@ -226,6 +223,14 @@ - (void) deleteItemFromDeviceByFileDto:(FileDto *) file {
         }
         
     } else {
+        ManageThumbnails *manageThumbnails = [ManageThumbnails sharedManager];
```

Comment:
We should use directly the singleton not getting the singleton in a local instance. Just to use the same style on all the app (like the other singletons) :wink: